test/tools/runner/filters/client_encryption_filter.js

```diff
@@ -40,11 +39,7 @@ class ClientSideEncryptionFilter {
     const clientSideEncryption =
       test.metadata && test.metadata.requires && test.metadata.requires.clientSideEncryption;
 
-    // CSFLE is only supported on LTS versions of node
```

Comment:
Question: Are we removing the node version check because we always test on versions of node >4?

Reply:
Yep!